dipp/tensorflow/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2014-2017 The DIPP contributors
+#
+# This file is part of DIPP.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Custom add-ons for tensorflow."""
```

Comment:
I thought that we could in these framework-specific `__init__.py` files try to import the framework and report an error immediately to the user, with instructions for installation. Otherwise the import will just fail at a random location.
```python
try:
    import tensorflow
except ImportError:
    raise ImportError('to use `dipp/tensorflow`, you need to install the `tensorflow`'
                      'package either from conda: `conda install tensorflow`, '
                      'or using pip: `pip install tensorflow`)
```
Sort of.

Reply:
Agree that we should do this